packages/generator-single-spa/package.json

```diff
@@ -8,7 +8,7 @@
     "src"
   ],
   "scripts": {
-    "test": "jest --testPathIgnorePatterns templates"
```

Comment:
I moved this to the jest config to support being able to run `yarn test basic.test.js`. Without it, it would actually append that arg to the `testPathIgnorePatterns` and it would actually run all the other tests instead of just the one you wanted.